app/src/main/java/com/example/dashcarr/presentation/tabs/map/MapFragment.kt

```diff
@@ -67,10 +58,13 @@ import java.time.format.DateTimeFormatter
 class MapFragment : BaseFragment<FragmentMapBinding>(
     FragmentMapBinding::inflate,
     showBottomNavBar = true
-), LocationListener, MapEventsReceiver, SensorEventListener {
+), LocationListener, MapEventsReceiver {
 
     private val viewModel: MapViewModel by viewModels()
     private val recordingViewModel: RecordingViewModel by viewModels()
+    private val sensorRecordingViewModel by lazy {
```

Comment:
Why are you not using `private val sensorRecordingViewModel by viewModels()`?

Reply:
We talked about this